src/main/java/ch/njol/skript/lang/SkriptEvent.java

```diff
@@ -65,7 +70,9 @@ public abstract class SkriptEvent extends Structure {
 	public final boolean init(Literal<?>[] args, int matchedPattern, ParseResult parseResult, EntryContainer entryContainer) {
 		this.expr = parseResult.expr;
 
-		EventPriority priority = getParser().getData(EventData.class).getPriority();
+		EventData eventData = getParser().getData(EventData.class);
```

Comment:
Is this variable used anywhere other than the line below?

Reply:
Yes, line 96.